src/Components/Bookings/Bookings.js

```diff
@@ -1,10 +1,182 @@
-import React from "react";
+import React, { useState, useContext, useEffect } from "react";
+import "./Bookings.css";
+import BookingsModal from "./BookingsModal/BookingsModal";
+import Backdrop from "./Backdrop/Backdrop";
+import makeRequest from ".././../Utils/index";
+import deleteBooking from "../.././Helpers/Delete";
+import editBooking from "../.././Helpers/Edit";
+import getCompletedTrips from "../../Helpers/completedTrip";
+import CompletedModal from "./CompletedTripModal/CompletedModal";
+import Authcontext from "../Context/context";
 
 const Bookings = (props) => {
+	const [modalViewing, setModalViewing] = useState(false);
+	const [bookings, setBookings] = useState([]);
+	const [selectedBooking, setSelectedBooking] = useState(null);
+	const [isEdit, setIsEdit] = useState(false);
+	const [completed, setCompleted] = useState(null);
```

Comment:
what's the difference between `setCompleted` and `setCompletedTrip`? Please use more descriptive names if they are different

Reply:
resolved